src/modules/components/proposal/proposalVoting/proposalVotingBodyContent/proposalVotingBodyContent.test.tsx

```diff
@@ -0,0 +1,152 @@
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { ProposalVotingStatus } from '../../proposalUtils';
+import { ProposalVotingStageContextProvider, type IProposalVotingStageContext } from '../proposalVotingStageContext';
+import { ProposalVotingBodyContent, type IProposalVotingBodyContentProps } from './proposalVotingBodyContent';
+
+describe('<ProposalVotingBodyContent /> component', () => {
+    const createTestComponent = (
+        props?: Partial<IProposalVotingBodyContentProps>,
+        contextValues?: Partial<IProposalVotingStageContext>,
+    ) => {
+        const defaultProps: IProposalVotingBodyContentProps = {
+            status: ProposalVotingStatus.PENDING,
+            name: 'Test Stage',
+            bodyId: 'body1',
+        };
+        const completeProps = { ...defaultProps, ...props };
+        const defaultContextValues: IProposalVotingStageContext = {
+            startDate: 0,
+            endDate: 0,
+        };
+        const contextValue = { ...defaultContextValues, ...contextValues };
```

Comment:
Same for other test files 🙏🏻 
```suggestion
        const completeContextValues: IProposalVotingStageContext = {
            startDate: 0,
            endDate: 0,
            ...contextValues,
        };
        const contextValue = { ...defaultContextValues, ...contextValues };
```